### PR TITLE
Location holdings report

### PR DESCRIPTION
**Pull request by marktriggs**

### CODE REVIEW

File: backend/app/model/reports/abstract_report.rb [hunk 1]
```diff
@@ -40,7 +40,7 @@ def processor
   end
 
   def query(db)
-    raise "Please specific a query to return your reportable results"
+    raise "Please specify a query to return your reportable results"
```
Comment (payten) [inline backend/app/model/reports/abstract_report.rb]: :rocket:

File: backend/app/model/reports/location_holdings_report.rb [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+      @start_location = Location.get_or_die(JSONModel(:location).id_for(params['location_start']['ref']))
+
+      if params['location_end'] && !params['location_end'].empty?
+	@end_location = Location.get_or_die(JSONModel(:location).id_for(params['location_end']['ref']))
+      end
+    end
+  end
+
+  def title
+    "Location Holdings Report"
```
Comment (marktriggs) [inline backend/app/model/reports/location_holdings_report.rb]: i18n

File: backend/app/model/reports/location_holdings_report.rb [hunk 3]
```diff
@@ -0,0 +1,268 @@
+class LocationHoldingsReport < AbstractReport
+
+  register_report({
+                    :uri_suffix => "location_holdings",
+                    :description => "Report on containers shelved at one or more locations",
+                    :params => [["locations", "LocationList", "The locations of interest"]]
```
Comment (payten) [inline backend/app/model/reports/location_holdings_report.rb]: I18n

File: backend/app/model/reports/location_holdings_report.rb (hunk 2 above)
Comment (payten) [inline backend/app/model/reports/location_holdings_report.rb]: I18n

**Pull request merged**